pydis_site/apps/content/resources/guides/python-guides/windows/common-issues.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Common Issues Using Python on Windows
+description: A list of common issues with Python on Windows and how to fix them
+icon: fab fa-windows
+toc: 3
+---
+
+### When I run `python` in the terminal I get no result, or the Windows Store opens!
```

Comment:
```suggestion
### When I run `python` in the terminal I get no result, or the Microsoft Store opens!
```
probably my mistake, but since I see it here. I write the wrong thing all the time :P